### PR TITLE
validate func calls & add tests

### PR DESCRIPTION
**Pull request by mrMakaronka**

### CODE REVIEW

File: pylzy/lzy/api/v1/__init__.py [hunk 1]
```diff
@@ -41,22 +41,23 @@
 # pylint: disable=[invalid-name]
 # noinspection PyShadowingNames
 def op(
-    func: Optional[FuncT] = None,
```
Comment (lindlind) [inline pylzy/lzy/api/v1/__init__.py:44]: new codestyle, use 4 spaces, plz

File: pylzy/tests/api/v1/test_calls.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+# noinspection PyUnusedLocal
+@op
+def one_arg(arg: int) -> str:
+    pass
+
+
+# noinspection PyUnusedLocal
+@op
+def one_default(arg: int, b: Optional[str] = None) -> str:
```
Comment (lindlind) [inline pylzy/tests/api/v1/test_calls.py:25]: I would test default arg without optional, and for Optional type I would create another test.

File: pylzy/tests/api/v1/test_calls.py [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+        # noinspection PyUnresolvedReferences
+        func2: FuncSignature = wf.owner.runtime.calls[1].signature.func
+        self.assertEqual(str, func2.output_types[0])
+        self.assertEqual(1, len(func2.input_types))
+        self.assertEqual(int, func2.input_types['arg'])
+
+    def test_default(self):
+        with self.lzy.workflow("test") as wf:
+            one_arg(1)
```
Comment (lindlind) [inline pylzy/tests/api/v1/test_calls.py:85]: one_default(1) ?

File: pylzy/tests/api/v1/test_calls.py [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+
+from pydantic import ValidationError
+
+from lzy.api.v1 import Lzy, op
+from lzy.api.v1.signatures import FuncSignature
+from lzy.api.v1.utils.proxy_adapter import materialized
+from tests.api.v1.mocks import RuntimeMock, StorageRegistryMock
+
+
+@op(description="aaa")
```
Comment (lindlind) [inline pylzy/tests/api/v1/test_calls.py:12]: Where is this description used in tests?

File: pylzy/tests/api/v1/test_calls.py [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+        self.assertEqual(6, len(func.input_types))
+        self.assertEqual(str, func.input_types['a'])
+        self.assertEqual(int, func.input_types['b'])
+        self.assertEqual(float, func.input_types['c'])
+        self.assertEqual(('a', 'b'), func.kwarg_names)
+
+    def test_multiple_return_values(self):
+        # noinspection PyUnusedLocal
+        @op
+        def multiple_return() -> (int, str):
```
Comment (lindlind) [inline pylzy/tests/api/v1/test_calls.py:214]: Why some ops are defined here, and some are higher?

**Review by lindlind — CHANGES_REQUESTED** [review]

File: pylzy/tests/api/v1/test_calls.py (hunk 5 above)
Comment (mrMakaronka) [reply pylzy/tests/api/v1/test_calls.py:217]: If ops are used multiple times they defined globally

File: pylzy/tests/api/v1/test_calls.py (hunk 4 above)
Comment (mrMakaronka) [reply pylzy/tests/api/v1/test_calls.py]: Removed it :)

File: pylzy/lzy/api/v1/__init__.py (hunk 1 above)
Comment (mrMakaronka) [reply pylzy/lzy/api/v1/__init__.py:44]: why are all codestyle checks are green in that case?

File: pylzy/tests/api/v1/test_calls.py (hunk 2 above)
Comment (mrMakaronka) [reply pylzy/tests/api/v1/test_calls.py]: agree

File: pylzy/tests/api/v1/test_calls.py (hunk 3 above)
Comment (mrMakaronka) [reply pylzy/tests/api/v1/test_calls.py]: yes, it was typo

File: pylzy/lzy/api/v1/__init__.py (hunk 1 above)
Comment (mrMakaronka) [reply pylzy/lzy/api/v1/__init__.py]: changed local codestyle settings

**Pull request merged**